### ISSUE TITLE
Further info needed for Digital Ocean setup

### ISSUE THREAD

**Issue by h4x0rm1k3** [xMNG/Microsoft-Rewards-Bot#44]
I've got it all running nicely in Windows 10 but i'd like to delve in to getting a server to do it for me and decided to use Digital oceans free trial as the bot's supposed to work with it. Anyway, I get as far as setting up a Ubuntu droplet and SSH in and login and changing the root password as it asks to. 
From there I clone your git and then I try to install and get errors that pip isn't installed and it can't continue any further and i've been stuck for a while now even after delving in to Google and i'm pretty good at working things out most of the time but this time i'm stumped so I was wondering if you have a list of all the commands that i'd need to enter to get the bot & all the requirements installed and how to edit the json file also as i'm not 100% sure about that either although I could probably work it out once past this issue.
Other than that i'm mightily impressed with the bot itself and can confirm it works perfectly for me in the UK and runs for over more than 3 accounts although it does throw up a lot of errors and such in the cmd window but i've noticed it doesn't matter as the log reports everything completed and on checking to be certain it's all confirmed complete there as well.

**Comment by h4x0rm1k3** [comment]
There is another issue i've noticed as well which is I can't get the batch file to work, it just quickly opens cmd and it closes again and nothing happens after that as no log is made etc. I've been trying to figure out how to get it to run automatically every day but so far haven't found a way even in Task scheduler so if anyone has any ideas then i'd be happy to hear them. I can run the command in the batch file fine in cmd but running the file does the above so am a bit lost, especially when it comes to using python as well.

**Comment by h4x0rm1k3** [comment]
I think i've cracked it! I'm not sure if it's a combination of issue 45 and linked in some way as I installed the driver manually using SFTP on the droplet adding relevant permissions but I still got the same error but hadn't noticed 1 of the error's i'd been getting were to do with Selenium and it being linked with Python 2.7 only! So, I ran the command sudo apt-get install python3-selenium and it installed everything which then allowed me to run the golden python3 ms_rewards.py and it ran fine so far with 1 error about chrome_options now just being options which i've changed in the script along with forcing --headless & --all without having to include them anymore. So, now all i've got to do is wait until about 2am GMT when the accounts I run roll over to 0 for the day and see how the script runs! I've also sorted the bat file running on my PC with task scheduler too, I had to edit the .bat file to just py instead of python followed by the full path to where the script is. Fingers crossed I can close this off as solved myself and will probably add a small guide for others with issues.
